frontend/src/metabase/query_builder/components/view/sidebars/QuestionSettingsSidebar/tests/setup.tsx

```diff
@@ -48,13 +50,20 @@ export const setup = async ({
     }),
   ]);
 
-  setupDatabaseEndpoints(
-    createSampleDatabase({
-      settings: {
-        "persist-models-enabled": dbHasModelPersistence,
-      },
-    }),
-  );
+  const database = createSampleDatabase({
+    settings: {
+      "persist-models-enabled": dbHasModelPersistence,
+    },
+  });
```

Comment:
Maybe you could do something similar do this, , so that you don't need to mutate `database`.

https://github.com/metabase/metabase/blob/8c5879803de6bf421952fafc076978a81d6f60ee/frontend/src/metabase/admin/databases/components/DatabaseModelFeaturesSection/DatabaseModelFeaturesSection.unit.spec.tsx#L138-L140

`createSampleDatabase` calls `createMockDatabase` under the hood, so I think it should work.